app/Console/Kernel.php

```diff
@@ -104,14 +104,15 @@ private function scheduleJobsForCollectionsOrGalleries(Schedule $schedule): void
                 ->hourlyAt(5);
 
         $schedule
-            ->command(FetchCollectionOpenseaSlug::class)
+            ->command(FetchWalletNfts::class)
             ->withoutOverlapping()
-            ->hourlyAt(10);
+            ->hourlyAt(10); // offset by 10 mins so it's not run the same time as FetchEnsDetails...
```

Comment:
Was there any specific reason to change the `FetchWalletNfts` schedule to `hourlyAt(10)` or would this PR work without the change just as well?

Just asking out of curiosity, fine with keeping the change regardless.